### ISSUE TITLE
1.4.1 breaks the plugin

### ISSUE THREAD

**Issue by robinscholz** [t-k-f/vue-packery-plugin#9]
Uupdating your plugin past 1.3.1 breaks my install. Not sure what exactly happens, but the container gets a height far greater than needed and all packery elements are set from the bottom up, even though they should be set from the top.

1.3.1 works for me, so the bug was introduced by the draggability update.

**Comment by jan-thoma** [comment]
i created examples with version 1.4.4 . which can be run with vue-cli3, these don't show the behaviour mentioned in this issue. Please provide a fiddle or example where this issue is present

**Comment by hesselberg** [comment]
Experiencing the same thing, although downgrading to to 1.3.1 doesn't seem to solve the issues.

For reference I'm using Nuxt, but not sure it matters.


https://jsfiddle.net/zygb0kuL/16/
( didn't seem to find a good way of importing your plugin into the fiddle )

**Comment by jan-thoma** [comment]
@hesselberg include the dist/build.js file in the fiddle

**Comment by hesselberg** [comment]
@jan-thoma apologies, I'm not savant at jsfiddle. I tried using the Resource options on the left side, but using `https://unpkg.com/vue-packery-plugin@1.4.6/dist/build.js` doesn't seem to help.

I also added `Vue.use('packeryPlugin')` at the end of the JS file, probably need to do more?
New fiddle: https://jsfiddle.net/oeq6rwgb/

Regardless, I hope the actual markup is enough for you to take a look at it :-)
